tool_collections/cufflinks/cuff_macros.xml

```diff
@@ -48,27 +54,25 @@
             #if $in_type.set_in_type in ['BAM', 'CXB']
                 #for $condition in $in_type.conditions:
                     #set samples = ','.join( [ str( $sample ) for $sample in $condition.samples ] )
-                    $samples
+                    "${ samples }"
                 #end for
             #elif $in_type.set_in_type == 'CONDITION_LIST'
                 #for $sample in $in_type.conditions:
-                    $sample
+                    "${ sample }"
                 #end for
             #elif $in_type.set_in_type == 'CONDITION_REPLICATE_LIST'
                 #for $condition_list in $in_type.conditions:
-                    #set samples = ','.join( [ str( $sample ) for $sample in $condition_list ] )
-                    $samples
+                    "${ ','.join( map( str, $condition_list ) ) }"
                 #end for
             #end if
   </token>
   <token name="@CONDITION_LABELS@">
-            #import re
             #if $in_type.set_in_type in ['BAM', 'CXB']
-                #set labels = '\'' + '\',\''.join( [ str( $condition.name ) for $condition in $in_type.conditions ] ) + '\''
+                --labels "${ ",".join( map( lambda x: str( x.name ).replace( ",", "_" ), $in_type.conditions ) ) }"
             #elif $in_type.set_in_type in ['CONDITION_LIST', 'CONDITION_REPLICATE_LIST']
-                #set labels = '\'' + '\',\''.join( map(lambda x: re.sub('[^\w\-_]', '_', x), $in_type.conditions.keys() ) ) + '\''
+                #import pipes
```

Comment:
pipes is deprecated since 2.7 isn't it? What does this mean if Galaxy at some point switches to Python3?